pkg/apis/api.acorn.io/v1/types.go

```diff
@@ -325,6 +325,7 @@ type Config struct {
 	PublishBuilders              *bool          `json:"publishBuilders" name:"publish-builders" usage:"Publish the builders through ingress to so build traffic does not traverse the api-server"`
 	BuilderPerNamespace          *bool          `json:"builderPerNamespace" name:"builder-per-namespace" usage:"Create a dedicated builder per namespace"`
 	InternalRegistryPrefix       string         `json:"internalRegistryPrefix" name:"internal-registry-prefix" usage:"The image prefix to use when pushing internal images (example ghcr.io/my-org/)"`
+	DefaultWorkloadMemory        int64          `json:"defaultWorkloadMemory" name:"default-workload-memory" quantity:"true" usage:"Set the memory limit default for installed Acorns." short:"m" default:"256M"`
```

Comment:
```suggestion
	DefaultWorkloadMemory        int64          `json:"defaultWorkloadMemory" name:"default-workload-memory" quantity:"true" usage:"Set the memory limit default for acorn workloads." short:"m" default:"256M"`
```

Suggesting this change in an attempt to clarify that this isnt the quantity for the entire acorn, but for each container in it.

Reply:
also, drop the word limit as it implies just the k8s "limit", but this is for limit and request. Basically, don tuse the limit qualifier